### DISCUSSION TITLE
Mobile scrollbar

### DISCUSSION THREAD

**Discussion by MyNameTony1000** [KingSora/OverlayScrollbars#502] [Q&A] [answered]
Hello) I noticed that in the mobile version or when using the "device toolbar" - the default scroll appears again on the page. You can see it especially well when you start changing the plugin slider color or size. Can you tell me if something can be done about it?

**Comment by KingSora** [comment, marked as answer]
@MyNameTony1000 Good day :)

Please post your initialization code.. There are some options which make the native scrollbar visible if it is overlaid. I implemented them so you can use this plugin for systems which don't have a native scrollbar and use the native on systems which have one.

Please also post your Operating System and browser version you are using.

**Reply by MyNameTony1000** [reply]
`this.data.pageScroll = OverlayScrollbars(document.body, { overflow: { x: 'hidden', }, scrollbars: { visibility: 'auto', autoHide: 'scroll', autoHideDelay: 300, }, });`

Windows 10, brave last version
Just now tried using the showNativeOverlaidScrollbars option - but it doesn't update the slider when switching emulation

**Reply by KingSora** [reply]
Ah you're initializing to the `body` element.. That element behaves a little bit more special.. In modern browsers there shouldn't be a difference though. In older browsers its possible that `OverlayScrollbars` won't initialize if it would alter the browser functionality (like pull to refresh on mobile or `window.scroll` not working anymore) with the initialization. At least thats the default behavior. You could change that behavior like this:

```js
this.data.pageScroll = OverlayScrollbars(
  { 
    target: document.body,
    cancel: {
      // false => never cancel the initialization
      // true => always cancel the initialization
      // null => only cancel the initialization if the browser functionality is affected (thats the default)
      body: false,
      // false => never cancel the initialization if native scrollbars are overlaid (thats the default)
      // true => always cancel the initialization if native scrollbars are overlaid
      nativeScrollbarsOverlaid: false,
    }
  }, 
  { overflow: { x: 'hidden', }, scrollbars: { visibility: 'auto', autoHide: 'scroll', autoHideDelay: 300, }, }
);
```

**Reply by MyNameTony1000** [reply]
It turns out that the settings are applied only during initialization and when switching the emulation, the bands will not be updated?

**Reply by KingSora** [reply]
What do you excatly mean with "switching the emulation"?

**Reply by MyNameTony1000** [reply]
when you use ''toggle device toolbar'' in your browser. To switch to the view of other devices. When I first turned it on, looked and returned to the normal version again

**Reply by KingSora** [reply]
You have to reload with each device because the plugin gathers its environment with the first initialization. In reality the environment cant change thus there isnt any possibility to refresh it.

**Reply by MyNameTony1000** [reply]
I understand, thanks a lot for your help.

**Reply by KingSora** [reply]
youre welcome!
